packages/components/src/toolbar/toolbar/style.scss

```diff
@@ -20,12 +20,6 @@
 
 .components-accessible-toolbar,
 .components-toolbar {
-
-	&[aria-orientation="vertical"] {
-		flex-direction: column;
-		align-items: center;
-	}
-
```

Comment:
Why did you move this @scruffian? I put it here so the toolbar component can be used vertically in any other setting, not just zoom out. Since we added the prop for it too, without the CSS there is no visual change if orientation is set to vertical.